deployability/modules/testing/tests/test_manager/test_stop.py

```diff
@@ -22,6 +22,7 @@ def wazuh_params(request):
         'targets': targets
     }
     yield params
+    logger.info('Restoring Manager status')
```

Comment:
the Wazuh manager

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5289/commits/7c1d2270fe036e068551866cd700baa9e1b5a2ea

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5289/commits/7c1d2270fe036e068551866cd700baa9e1b5a2ea